gloo/transport/tcp/tls/openssl/dynamic_library.h

```diff
@@ -0,0 +1,16 @@
+struct DynamicLibrary {
+  DynamicLibrary(const DynamicLibrary&) = delete;
+
+  void operator=(const DynamicLibrary&) = delete;
+
+  DynamicLibrary(const char* name, const char* alt_name);
+
+  void* sym(const char* name);
+
+  ~DynamicLibrary();
+
+private:
+  const char* lib_name;
```

Comment:
Do not store raw string array pointers, since its lifetime is not defined
```suggestion
  const std::string lib_name;
```

Reply:
addressed